lib/cartoRenderer.js

```diff
@@ -1073,10 +1073,32 @@ var cartoSelectorIsMatch = function(definition, feature, source, zoom) {
     for (var filterKey in definition.filters) {
       var filter = definition.filters[filterKey];
       
-      if (filter.op === "=") {
+      switch (filter.op) {
+      case "=":
         if (feature.properties[filter.key] !== filter.val) {
           return false;
         }
+        break;
+      case "<":
+        if (feature.properties[filter.key] >= filter.val) {
```

Comment:
Yeah, that’s what I realized right after posting. Too distracted this morning, I guess. Also been a while since I wrote this :P

I think encapsulating each of the kinds of tests in this method might be better for reading, though it would be slower. Either way, not really in the scope of work for this PR. My bad.